circleci/images/citusupgradetester/files/etc/requirements.txt

```diff
@@ -9,7 +9,6 @@ certifi==2019.9.11
 cffi==1.12.3
 click==6.7
 construct==2.9.45
-cryptography==2.3.1
```

Comment:
We can't just update the requirements here because the base image is different and it isn't trivial to update it. So I just removed the vulnerable dependency as we don't use it instead of spending time for updating a package that isn't used.

Currently we have one venv for all the tests on citus side but we could have one per test so that we wouldn't have this kind of problems.